src/inc/msquic.h

```diff
@@ -716,7 +717,8 @@ typedef struct QUIC_SETTINGS {
             uint64_t HyStartEnabled            : 1;
 #ifdef QUIC_API_ENABLE_PREVIEW_FEATURES
             uint64_t EncryptionOffloadAllowed  : 1;
-            uint64_t ReservedFlags             : 62;
+            uint8_t ReliableResetEnabled       : 1;
```

Comment:
```suggestion
            uint64_t ReliableResetEnabled      : 1;
```
match the types of everything else in the bitfield